README.md

```diff
@@ -78,7 +78,7 @@ introspection and control.
 
 # Install
 
-See the [installation tutorial](https://gazebosim.org/api/sim/9/install.html).
+See the [installation tutorial](https://gazebosim.org/docs/latest/getstarted/).
```

Comment:
I'd rather not completely get rid of the other link since it might be useful for folks who are using libgz-sim as a library.  Maybe we can say, 
```suggestion
For installing Gazebo, see the [getting started guide](https://gazebosim.org/docs/latest/getstarted/). If you want to use `libgz-sim` as a library, see the [installation tutorial](https://gazebosim.org/api/sim/9/install.html)
```